### DISCUSSION TITLE
Pressing "return" on signin menu focuses the next input field.

### DISCUSSION THREAD

**Discussion by dmitrin9** [monkeytypegame/monkeytype#6051] [Ideas]
I vaguely remember this feature existing on some linux distros? (correct me if I'm wrong).

It's just that whenever I try to sign in, I need to manually click with my mouse in order to go to the next input field. It just feel way more smooth to press return to move to the next input field. It seems like a random tiny change that wouldn't mean anything, but in my opinion it would make the overall signin UX a lot more seamless.

Thank you for considering my request. :)
Warm regards.

PS: I'm also curious to hear if implementing a feature like this would introduce any vulnerabilities or security bugs?

**Comment by Miodec** [comment]
You can just press tab to move to the next input field.
